thorlcr/thorutil/thormisc.hpp

```diff
@@ -340,7 +340,7 @@ class CFileSizeTracker: public CInterface
 // simple class which takes ownership of the underlying file and deletes it on destruction
 class graph_decl CFileOwner : public CSimpleInterface, implements IInterface
 {
-    OwnedIFile iFile;
+    IFileAttr iFile;
```

Comment:
had to look up what a 'IFileAttr' was/did (I see it's only ever been used in 1 place before).
IMO it's less clear than the obvious semantics if it read  "Linked<IFile> iFile", with no upside.

Reply:
I agree.  I'm not sure why we have OwnedIFile either.

Reply:
yeah, but at least that one explicitly conveys the semantics.